pkgs/development/python-modules/django-currentuser/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, django
+, hatchling
+}:
+let
+  version = "0.6.1";
+in
+buildPythonPackage {
+  pname = "django-currentuser";
+  inherit version;
+  format = "pyproject";
```

```suggestion
  pyproject = true;
```
Please use `pyproject = true` instead.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

---

Done!